pkg/services/postgresqldb/types_all_in_one.go

```diff
@@ -5,10 +5,8 @@ import "github.com/Azure/open-service-broker-azure/pkg/service"
 // AllInOneProvisioningParameters encapsulates both dbms and database
 // PostgreSQL-specific provisioning options
 type AllInOneProvisioningParameters struct {
-	SSLEnforcement  string   `json:"sslEnforcement"`
-	FirewallIPStart string   `json:"firewallStartIPAddress"`
-	FirewallIPEnd   string   `json:"firewallEndIPAddress"`
-	Extensions      []string `json:"extensions"`
+	ServerProvisioningParameters
+	DatabaseProvisioningParameters
```

Comment:
This is sweet.

Wondering if we want to change `ServerProvisioningParameters` to `DMSProvisioningParameters`. Dealer's choice.

Reply:
Oh. I see your rationale for this in another comment.

Reply:
I might want to revisit this in the future, but it's certainly not blocking this PR.